src/components/Checkbox/Checkbox.js

```diff
@@ -13,7 +13,7 @@ const Checkbox = (props) => {
         type="checkbox"
         id={elementId}
         checked={props.checked}
-        onChange={props.onChange}
+        onChange={e => props.onChange(e.target.checked)}
```

Comment:
Updated this to match the new `<Toggle>` component which needs to send the actual value of `checked` to save in local storage (rather than just calling some 'toggle' function).